skimage/measure/tests/boundary_fixtures.py

```diff
@@ -0,0 +1,2060 @@
+import numpy as np
+
+HORSE_EXPECTED = np.array(
```

Comment:
Why not save the array in a `.npz` file (compressed or uncompressed) using `np.save("expected_boundary_fixtures_horse.npy", HORSE_EXPECTED)`? This will create a much smaller file and much cleaner PR.

You can use it in your test as
```
HORSE_EXPECTED = np.load("expected_boundary_fixtures_horse.npy")
```

Reply:
Oh, that's neat! I didn't know it was possible and I'll definitely use it. Thanks!